### PR TITLE
Added deployment configs for AWS UPI with RHEL workers

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
+  rhel_workers: true
+  rhcos_ami: 'ami-06c85f9d106577272'
+  rhel_worker_ami: 'ami-073251e3a0e36cebb' # RHEL7.7 as of now
+  rhel_worker_instance_type: 'm5.4xlarge'
+  root_disk_size: 50
```
Comment (shylesh) [inline conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml]: Coady, according Jira requirements root disk size should be 120GB.

Comment (petr-balogh) [reply conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml]: https://jira.coreos.com/browse/KNIP-658  @vavuthu  BTW do we following this also for VmWare? Thanks

Comment (vavuthu) [reply conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml]: Its 60GB. I will submit PR for changing to 120 GB

Comment (clacroix12) [reply conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml]: Done

Comment (clacroix12) [reply conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml]: Done

**Comment by clacroix12** [comment]
@petr-balogh Attempts at getting a PR test to pass have been failing all day, however I was able to successfully deploy with these configurations locally. The PR job has failed multiple times for reasons unrelated to these configurations. In addition no jobs will be utilizing these configurations immediately so I think it is safe to merge these.

**Pull request merged**